src/foremast/utils/awslambda.py

```diff
@@ -99,22 +100,59 @@ def add_lambda_permissions(function='',
     session = boto3.Session(profile_name=env, region_name=region)
     lambda_client = session.client('lambda')
     response_action = None
+    prefixed_sid = 'foremast-' + statement_id
```

Comment:
Better, but there are still two `foremast-` string locations we have to maintain.